content/blog/building-a-future-of-cloud-engineering/index.md

```diff
@@ -54,12 +54,12 @@ At the same time, "DevOps" as an approach to organizing teams and
 developing software has matured greatly, and is now the default for many
 organizations. New practices such as infrastructure-as-code, immutable
 infrastructure, and Site Reliability Engineering have modernized these
-roles, giving us approaches that can leverage more rigorous software
-engineering practices to tasks that used to be more ad-hoc, manual, and
+roles, giving us approaches that can leverage a more rigorous software
+engineering experience to tasks that used to be more ad-hoc, manual, and
```

Comment:
This doesn't quite say the same thing.

Reply:
That one is probably fine to leave as it is then. Unless "can leverage a better experience through more rigorous software engineering practices..." works? It's not a super crucial one, but "experience" kept coming up in our semantic analysis.